### DISCUSSION TITLE
Support for sass variables in the online editor?

### DISCUSSION THREAD

**Discussion by kaoskeya** [bootstrap-email/bootstrap-email#144] [Help]
Hi, just purchased the templates and they have already saved me a lot of time! One point though, it would be great if we could specify the primary, secondary colors, etc while using the online editor.

Thanks for all the work!

**Comment by stuyam** [comment]
100% I want to add this feature it is definitely on my list of things to do! Thanks for the feedback and thanks for the support it means a lot ❤️
